language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/FunctionCallExpressionNodeContext.java

```diff
@@ -84,4 +93,26 @@ public void sort(BallerinaCompletionContext context,
                     .setSortText(SortingUtil.genSortTextByAssignability(completionItem, symbol));
         }
     }
+
+    private List<LSCompletionItem> getNamedArgExpressionCompletionItems(BallerinaCompletionContext context,
```

Comment:
In the usage, the logic for capturing the parameters has been duplicated. We can unify this

Reply:
Done